### PR TITLE
Fixes #227 enableSourceMaps is not working in firefox

### LINKED ISSUE #227: Weird request for map
In FireFox Im seeing mangled requests to main.map in the console.  Works ok in Chrome.  According to the angular people, this is coming from ngx-logger.  Not a show stopped but strange.

See https://github.com/angular/angular-cli/issues/20085

### PR DESCRIPTION
**Pull request by bmtheo**
Actually it fixes the stackLine parsing when using firefox

This fixes the sourceMap location and the LogPostion

### CODE REVIEW

File: src/lib/utils/logger.utils.ts [hunk 1]
```diff
@@ -39,32 +39,6 @@ export class NGXLoggerUtils {
     return configColorScheme[level];
   }
```
Comment (qortex) [inline src/lib/utils/logger.utils.ts:41]: Isn't this function still necessary?

**Review by qortex — COMMENTED** [review]
Testing looks good, thanks!
The test for Firefox seems good enough, '@' character being relatively rare.

File: src/lib/utils/logger.utils.ts (hunk 1 above)
Comment (bmtheo) [reply src/lib/utils/logger.utils.ts:41]: Not sure what you are pointing out, it seems like you referenced `getColorFromConfig` func but I did not touch it and yes it is used by `getColor`

Comment (qortex) [reply src/lib/utils/logger.utils.ts:41]: Github shows that the function `getCallerDetails` is now removed in the commit

Comment (bmtheo) [reply src/lib/utils/logger.utils.ts:41]: yes there was 2 of those, one static that was not used anymore and I deleted it. The one used is still there in mapper.service.ts

Comment (qortex) [reply src/lib/utils/logger.utils.ts:41]: Ok cool

**Review by qortex — APPROVED** [review]
Ok

**Pull request merged**